### PR TITLE
DE2908 - GO Local email changes username warning

### PR DESCRIPTION
**Pull request by tallented**
The Crossroads and Anywhere profile forms display a warning that changing email will change the username.

revised - email warning below - crossroads profile

revised - email warning below - anywhere profile

The Archdiocese and Other organization profile forms do not display the email warning

revised - email warning below - other profile

### CODE REVIEW

File: crossroads.net/app/go_volunteer/formly_wrappers/email_change_warning.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="row" ng-class="options.templateOptions.row.className">
+  <div ng-class="{ 'col-sm-6': $root.username != undefined, 'col-xs-12': $root.username == undefined }">
```
Comment (Ganonside) [inline crossroads.net/app/go_volunteer/formly_wrappers/email_change_warning.html]: should use triple equals

File: crossroads.net/app/go_volunteer/formly_wrappers/email_change_warning.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <formly-transclude></formly-transclude>
+  </div>
+  <div class="col-sm-6 soft-top mobile-hard-top" ng-if="$root.username != undefined">
```
Comment (Ganonside) [inline crossroads.net/app/go_volunteer/formly_wrappers/email_change_warning.html]: as above

File: crossroads.net/app/go_volunteer/page/profilePage/goVolunteerProfile.template.html [hunk 3]
```diff
@@ -35,7 +35,10 @@
       </ng-messages>
     </div>
 
-    <div class="form-group col-xs-12" ng-class="{'has-error': volunteerProfile.validate('email')}">
+  </div>
+  <div class="row">
+
+    <div class="form-group" ng-class="{ 'col-sm-6': $root.username != undefined, 'col-xs-12': $root.username == undefined, 'has-error': volunteerProfile.validate('email')}">
```
Comment (Ganonside) [inline crossroads.net/app/go_volunteer/page/profilePage/goVolunteerProfile.template.html]: triple equals

File: crossroads.net/app/go_volunteer/page/profilePage/goVolunteerProfile.template.html [hunk 4]
```diff
@@ -59,6 +62,13 @@
       </ng-messages>
     </div>
 
+    <div class="col-sm-6 soft-top mobile-hard-top" ng-if="$root.username != undefined">
```
Comment (Ganonside) [inline crossroads.net/app/go_volunteer/page/profilePage/goVolunteerProfile.template.html]: triple equals

File: crossroads.net/app/go_volunteer/formly_wrappers/email_change_warning.html (hunk 1 above)
Comment (tallented) [inline crossroads.net/app/go_volunteer/formly_wrappers/email_change_warning.html]: The triple equals does not actually behave as expected here.  Leaving as is with != correctly hides the DIV when user is not logged in.  Changing to !== incorrectly displays the warning message even when not logged in.

**Pull request merged**